oracle/README.md

```diff
@@ -321,6 +322,43 @@ Verifies the database is available and accepting connections.
 
 ## Troubleshooting
 
+### Common Problems
+#### Oracle Instant Client
+- Verify that both the Oracle Instant Client and SDK files are located in the same directory.
+
+##### Linux
+- See further installation documentation on [Oracle][15].
+
+##### Windows
+- Verify the Microsoft Visual Studio <YEAR> Redistributable requirement is met for your version, see [Windows downloads page][16] for more details.
+- Ensure the `PATH` environment variable contains `C:\oracle\instantclient_19_9`. 
```

Comment:
Are we sure the `PATH` env var is required for the agent?

Reply:
Might not always be `C:\oracle\instantclient_19_9` as the user has to decompress the archives in any directory. So this path can be anything

Reply:
Updated with example directory and used that as references